website/docs/d/resource_pool.html.markdown

```diff
@@ -0,0 +1,66 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_resource_pool"
+sidebar_current: "docs-vcd-data-source-resource-pool"
+description: |-
+  Provides a data source for a resource pool attached to a vCenter.
+---
+
+# vcd\_resource\_pool
+
+Provides a data source for a resource pool attached to a vCenter. A resource pool is an essential component of a Provider VDC.
+
+
+~> Note 1: this data source requires system administrator privileges
```

Comment:
For easier eye-scanning (similar to https://registry.terraform.io/providers/vmware/vcd/latest/docs/resources/nsxt_edgegateway):

```suggestion
~> Note 1: this data source requires System Administrator privileges
```

Reply:
Changed